frontend/src/routes/users/me/+page.svelte

```diff
@@ -0,0 +1,252 @@
+<script lang="ts">
+  import user from '$stores/user';
+  import wretch from 'wretch';
+
+  const api = wretch('http://localhost:3000/api');
+
+  let birthdate: string;
+  let isSuccess = false;
+  $: console.log(birthdate);
+  $: console.log(isSuccess);
```

Comment:
Let's avoid outputting development information to production. The console should be clean if it's possible